components/dialog/stories/dialog.stories.js

```diff
@@ -61,15 +84,37 @@ export default {
 			control: "boolean",
 		},
 		isOpen,
+		hasHeroImage: {
+			name: "Has hero image",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Content",
+			},
+			control: "boolean",
+			if: { arg: "layout", eq: "default" },
+		},
```

Comment:
Instead of adding a boolean to the UI, do we just want to assume that if `heroImageUrl` has content in the template that hasHeroImage is true?

Reply:
Yes! https://github.com/adobe/spectrum-css/pull/2833/commits/4c1cec586a4721b37931dacbfa78fa105e91d65b